Sources/MongoSwift/MongoCollection.swift

```diff
@@ -87,8 +98,8 @@ public class MongoCollection<T: Codable> {
     /// Drops this collection from its parent database.
```

Comment:
I forgot to mention before, we should also update this docstring so that it includes descriptions of the parameters. here's an [example](https://github.com/mongodb/mongo-swift-driver/blob/711a99ea5786ddcf056688b3100bffd457fd76bf/Sources/MongoSwift/MongoCollection%2BWrite.swift#L5) of a docstring where we document parameters

I noticed we never updated this to include the `ClientSession` parameter so I would just document that as something like "An optional session to use for this command".